### PR TITLE
adjusted regex and placement of alert

### PR DESCRIPTION
**Pull request by shewood**
### Description

List of proposed changes:
updated the regex for the join room user
fixed up the spacing of the error messages.

-
-

### CODE REVIEW

File: pages/home.js [hunk 1]
```diff
@@ -65,7 +65,7 @@ export default function Home(props) {
         throw t.invalidNameError
       }
       //Check if name contains special characters
-      else if (!/^[a-zA-Z0-9]+$/.test(username)) {
+      else if (!/^([A-Za-z0-9\s\-\'?])+$/.test(username)) {
```
Comment (DaceyTom2) [inline pages/home.js:68]: The regex should be pulled from a common spot between both fields that use it

File: components/ErrorLabel.js [hunk 2]
```diff
@@ -6,7 +6,9 @@ import PropTypes from 'prop-types'
 export function ErrorLabel(props) {
   return (
     <div
-      className={`border-l-4 border-[#D3080C] mt-2 sm:-mb-6 px-10 py-2 bg-[#F3E9E8] font-body font-bold text-sm lg:text-p ${props.className}`}
+      className={`border-l-4 border-[#D3080C] mt-2 sm:-mb-6 px-10 py-2 md:mb-0 bg-[#F3E9E8] font-body font-bold text-sm lg:text-p ${props.className}`}
+      id="error1"
```
Comment (DaceyTom2) [inline components/ErrorLabel.js:10]: It is possible to have error label on both create and join rooms.
Perhaps something more specific like joinRoomError, createRoomError

File: components/TextInput.js [hunk 3]
```diff
@@ -15,6 +15,8 @@ export default function TextInput(props) {
       <input
         type="text"
         aria-required="true"
+        aria-describedby="error1"
```
Comment (DaceyTom2) [inline components/TextInput.js:18]: Same issue here as above

**Review by DaceyTom2 — CHANGES_REQUESTED** [review]

File: components/TextInput.js (hunk 3 above)
Comment (shewood) [reply components/TextInput.js]: fixed it

File: components/ErrorLabel.js (hunk 2 above)
Comment (shewood) [reply components/ErrorLabel.js]: fixed it

File: pages/home.js (hunk 1 above)
Comment (shewood) [reply pages/home.js:68]: I will leave this for now.

**Review by DaceyTom2 — APPROVED** [review]

**Review by P0NDER0SA — APPROVED** [review]
looks ok to me!

**Pull request merged**